packages/dds/tree/src/core/tree/pathTree.ts

```diff
@@ -59,6 +60,30 @@ export interface FieldUpPath<TUpPath extends UpPath = UpPath> {
 	readonly field: FieldKey; // TODO: Type information, including when in DetachedField.
 }
 
+/**
+ * An UpPath at the root of a detached tree.
+ */
+export interface DetachedUpPath extends UpPath {
+	/**
+	 * The DetachedNodeId of this path.
```

Comment:
It's a bit weird to speak of the path having a detached node ID since only the root is detached/has a `DetachedNodeId`.
```suggestion
	 * The ID associated with this detached root node.
```